Python/Product/Uwp/Templates/Projects/BackgroundService/PythonBackgroundService.pyproj

```diff
@@ -55,4 +55,9 @@
 
   <Import Project="$(PtvsTargetsFile)" />
 
+  <PropertyGroup>
+    <TargetPlatformVersion>$winsdkversion$</TargetPlatformVersion>
+    <TargetPlatformMinVersion>$winsdkminversion$</TargetPlatformMinVersion>
+  </PropertyGroup>
+
```

Comment:
Ok, found the issue. The cpython.targets file (from https://github.com/ms-iot/python) was importing $(VCTargetsPath)\Application Type\Windows Store\Default.props which it doesn't need to. The target version should never be "7.0". By removing that import I can add WindowsTargetPlatformVersion and WindowsTargetPlatformMinVersion at the top of the user projects (this value gets assigned to TargetPlatformVersion and TargetPlatformMinVersion). This will actually match the RTM UWP projects for C# and C++.